.github/workflows/build-gtk.yml

```diff
@@ -0,0 +1,69 @@
+name: Build Controls.Sample.Gtk
+on: [push, pull_request]
+
+jobs:
+  build_and_test:
+    name: Build & Test
+    runs-on: ubuntu-22.04
+    env:
+      GtkSharpVersion: 3.24.24.117-develop
+      GtkSharpManifestVersion: 8.0.200
+      DotnetVersion: 8.0.200
+      PROJECTS_TO_PACK: |
+        src/BlazorWebView/src/Maui/Microsoft.AspNetCore.Components.WebView.Maui.csproj
+        src/Essentials/src/Essentials.csproj
+        src/Controls/src/Core/Controls.Core.csproj
+        src/Controls/src/Xaml/Controls.Xaml.csproj
+        src/Core/src/Core.csproj
+        src/Compatibility/Core/src/Compatibility.csproj
+        src/Controls/Foldable/src/Controls.Foldable.csproj
+    steps:
+      - name: Checkout MAUI repo
+        uses: actions/checkout@v2
+      - name: Setup .NET SDK ${{ env.DotnetVersion }}
+        uses: actions/setup-dotnet@v1.7.2
+        with:
+          dotnet-version: ${{ env.DotnetVersion }}
+      - name: Install gtk workload
+        run: |
+            # For some reason automatic workload manifest detection doesn't work (see https://github.com/GtkSharp/GtkSharp/issues/355#issuecomment-1446262239), so download and uzip mainfest file manually
+            dotnet nuget add source --name nuget.org "https://api.nuget.org/v3/index.json"
+            wget https://www.nuget.org/api/v2/package/gtksharp.net.sdk.gtk.manifest-${{ env.GtkSharpManifestVersion }}/$GtkSharpVersion -O gtksharp.net.sdk.gtk.manifest-${{ env.GtkSharpManifestVersion }}.nupkg            
+            DOTNET_DIR=/home/runner/.dotnet
+            WORKLOAD_MANIFEST_DIR=$DOTNET_DIR/sdk-manifests/${{ env.DotnetVersion }}/gtksharp.net.sdk.gtk
+            mkdir -p $WORKLOAD_MANIFEST_DIR
+            unzip -j gtksharp.net.sdk.gtk.manifest-${{ env.GtkSharpManifestVersion }}.nupkg "data/*" -d $WORKLOAD_MANIFEST_DIR/
+            rm gtksharp.net.sdk.gtk.manifest-${{ env.GtkSharpManifestVersion }}.nupkg
+            chmod 764 $WORKLOAD_MANIFEST_DIR/*
+            dotnet workload search
+            dotnet workload install gtk --skip-manifest-update
```

Comment:
@webwarrior-ws this is too big, move it to a script please

Reply:
done